guides/doc-Provisioning_Guide/topics/Virt-Libvirt.adoc

```diff
@@ -210,7 +224,7 @@ The compute profile CLI commands are not yet implemented in {ProjectName} {Produ
 
 In {Project}, you can use KVM provisioning to create hosts over a network connection and from an existing image.
 
-If you create a host with an existing image, the new host entry triggers the KVM server to create the virtual machine, using the pre-existing image as a basis for the new volume.
+If you create a host using an image provisioning method, the new host entry triggers the KVM server to create the virtual machine, using the selected pre-existing image as a basis for the new volume. Alternatively, if a Finish template is associated to the host operating system, {Project} connects to the VM using SSH protocol and executes the script. This can only work if DHCP server is under {Project} control, or IP address of the provisioning interface was manually set to the IP address of the virtual host.
```

Comment:
Again, new information not really appropriate for this PR, not upstreamising.

Reply:
Going to split this.